crates/sui-core/src/signature_mpc/mod.rs

```diff
@@ -243,7 +248,11 @@ impl SignatureMPCAggregator {
         message: SignatureMPCMessage,
     ) {
         let session_id = message.summary.session_id;
-        let sender_party_id = (epoch_store.committee().authority_index(&message.summary.auth_sig().authority).unwrap() + 1) as PartyID;
+        let sender_party_id = (epoch_store
+            .committee()
+            .authority_index(&message.summary.auth_sig().authority)
+            .unwrap()
```

Comment:
[It existed in the `sign-ia-wasm` branch](https://github.com/dwallet-labs/dwallet-network/blob/a2229184d2787cc6a945c071f6e30055082d51f2/crates/sui-core/src/signature_mpc/mod.rs#L251), and we decided to avoid refactoring existing code in this PR. Created an [issue](https://github.com/dwallet-labs/dwallet-network/issues/134) and added todo anyway.